src/shared/constants.ts

```diff
@@ -1,2 +1,3 @@
+export const MAX_CSV_UPLOAD_SIZE = 200000 // 0.2 MB
```

Comment:
nit, can we use standardise on using the binary size 1024 instead of 1000? would be good to rewrite this as either `200*1024 // 200KB` or `0.2*1024*1024 // 0.2MB`